src/components/Menu/MenuItem/MenuItemButton/MenuItemButton.tsx

```diff
@@ -44,7 +46,35 @@ export const MenuItemButton: FC<MenuItemButtonProps> = ({
     },
   ]);
 
-  return (
+  return secondaryButtonProps ? (
+    <li role="menuitem" tabIndex={tabIndex} className={menuItemClasses}>
+      <span className={styles.menuSecondaryWrapper}>
+        <button
```

Comment:
Was tabbing around and noticed there are limited styles for this raw button, and the keyboard styles are not matching the rest of the system. At the time this component was created, the `SystemUIButton` didn't exist. This one should prob be replaced in a future change with `SystemUIButton` with a slight modification to its text color.

Reply:
will move to SystemUIButton in the future